enterprise/server/backends/distributed/distributed.go

```diff
@@ -706,9 +706,8 @@ func (c *Cache) FindMissing(ctx context.Context, resources []*rspb.ResourceName)
 				}
 				for _, r := range resources {
 					hash := r.GetDigest().GetHash()
-					if _, ok := peerMissingHashes[hash]; !ok {
-						foundMap[hash] = struct{}{}
-					}
+					_, ok := peerMissingHashes[hash]
```

Comment:
change `ok` to `missing` to make this clearer

Reply:
Done.